### ISSUE TITLE
Enhancement: Get rid of "ON/OFF" button

### ISSUE THREAD

**Issue by Kampii** [DigitalSlideArchive/openseadragon-annotations#1]
Each tool button could have a code that disables the "zoom on clicks", thus, if a button is clicked then zoom is disabled. In order to stop drawing we would un-click the tool button.

**Comment by dgutman** [comment]
Agreed--- I do not like/love the way the events click on/off.  For example I think the shapes that can "self end" such as rectangle and point should basically auto "unclick" as you suggest.  I am going to be spending a lof of time refactoring this plugin in the near future.... please keep posting comments/bugs.
